test/resque_test.rb

```diff
@@ -9,6 +9,14 @@
     Resque.redis = @original_redis
   end
 
+  test "can push an item that depends on redis for encoding" do
```

Comment:
This is causing the travis failure. It should be `it`, not `test`, like the test right below it